src/objects/Application__c.object

```diff
@@ -0,0 +1,263 @@
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>false</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Submitted</fullName>
+                    <default>false</default>
+                    <label>Submitted</label>
+                </value>
+                <value>
+                    <fullName>Incomplete</fullName>
+                    <default>false</default>
+                    <label>Incomplete</label>
+                </value>
+                <value>
+                    <fullName>Received</fullName>
+                    <default>false</default>
+                    <label>Received</label>
+                </value>
+                <value>
+                    <fullName>In Review</fullName>
+                    <default>false</default>
+                    <label>In Review</label>
+                </value>
+                <value>
+                    <fullName>Waitlist</fullName>
+                    <default>false</default>
+                    <label>Waitlist</label>
+                </value>
+                <value>
+                    <fullName>Admit</fullName>
+                    <default>false</default>
+                    <label>Admit</label>
+                </value>
+                <value>
+                    <fullName>Deny</fullName>
+                    <default>false</default>
+                    <label>Deny</label>
+                </value>
+                <value>
+                    <fullName>Deferred Offer</fullName>
+                    <default>false</default>
+                    <label>Deferred Offer</label>
+                </value>
+                <value>
+                    <fullName>Accepted Offer</fullName>
+                    <default>false</default>
+                    <label>Accepted Offer</label>
+                </value>
+                <value>
+                    <fullName>Declined Offer</fullName>
+                    <default>false</default>
+                    <label>Declined Offer</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Application_Type__c</fullName>
+        <description>The type of program or track the applicant is applying to. For example, Undergraduate, Graduate, or Transfer.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The type of program or track the applicant is applying to. For example, Undergraduate, Graduate, or Transfer.</inlineHelpText>
+        <label>Application Type</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>false</restricted>
```

Comment:
@lifewithryan @xinnanlisf Everywhere else that we have picklist fields marked as unrestricted, the `restricted` tag simply doesn't exist in the XML. I'm going to remove it from these picklist fields.